### ISSUE TITLE
NaN when using Attribute

### ISSUE THREAD

**Issue by capandcamera** [kalkih/mini-graph-card#696]
In v0.10 and the latest dev release (at the time of this post) v0.11.0-dev5 I am struggling to retrieve data from the attribute of a sensor.

This attribute can display in a button card and an apex chart but shows as NaN in mini graph card (See below)
<img width="238" alt="Screenshot 2022-01-17 at 20 20 30" src="https://user-images.githubusercontent.com/79606905/149832687-c0f805c3-1ab4-4992-bd6f-95830ffe8ec9.png">

I've also attached a screenshot of the configurations of all 3 elements displaying this attribute
(the state_display is the button card config)
<img width="454" alt="Screenshot 2022-01-17 at 20 20 40" src="https://user-images.githubusercontent.com/79606905/149832706-0dd786be-77d6-4044-8d3d-b22cd37c3d12.png">
<img width="574" alt="Screenshot 2022-01-17 at 20 20 49" src="https://user-images.githubusercontent.com/79606905/149832723-943101cc-376a-4120-bc74-28f79b8ad849.png">

This suggests i've got the correct attribute name but for good measure i've also attached a screenshot of the states page.

<img width="1052" alt="Screenshot 2022-01-17 at 20 23 58" src="https://user-images.githubusercontent.com/79606905/149833032-0e91f9b6-ba07-4dcf-b711-00f3198e8b04.png">

Note: I'm using a decluttering card so [[container]] resolves to homeassistant so the entity is sensor.docker_homeassistant.

Thanks!

**Comment by jlsjonas** [comment]
Hey, thanks for the report!
I'm trying to reproduce the issue, but I'm sadly unable to do so.
To reduce variables, could you please try outside the decluttering card (thus also explicitly using `sensor.docker_homeassistant`), and may I ask what browser you're using?

Please note that attribute reporting has only been introduced in the beta, a `NaN` or similar would be expected on 0.10.0.

Finally, please ensure your (browser) cache is cleared after installing the 0.11.0-dev.5 beta, to ensure it's not a caching-related issue.

**Comment by capandcamera** [comment]
Hi, 

Great news, it works!

I was previously trying to reload the resources via service, but seems this wasn't doing a full reload.

I updated to 0.11.0-dev5 again, reloaded resources via service followed by a browser cache reset in Safari and Vivaldi (Chromium-Based) and both work now, 

Thank you for your help.

**Comment by luebbe** [comment]
Hi there,
it looks like I'm facing the same problem. As far as I can tell, I have v0.11.0 of mini-graph-card installed (where can I see this reliably in HACS?)
I have restarted HA and I have cleared the browser cache.
My goal is to display the `sensor.rpi_monitor_pi_anbau` `load_*_prcnt` attributes availble via MQTT. The developer functions show these values:
![grafik](https://user-images.githubusercontent.com/3872576/156035086-be87ebc5-bc2d-4a03-a08e-312412713c9c.png)

Here's the config and the NAN result.
![grafik](https://user-images.githubusercontent.com/3872576/156035192-f2680dc4-6cf7-48e9-a53b-68a245f4dfb0.png)

What can I do to resolve the problem

**Comment by jlsjonas** [comment]
@luebbe it does look like you're somehow not running 0.11 yet, if you see NaN for extra states when you add `
    show_state: true` to your entities I'll be even more sure.

If you click on "information" you should see the downloaded version at the top
<img width="152" alt="image" src="https://user-images.githubusercontent.com/932193/156915831-b9d8798b-17eb-4470-b56f-cd33f8c4e8b7.png">

Consider reinstalling the plugin if the issue persists

**Comment by luebbe** [comment]
Thanks for the reply. 
You mean `show_state: true` like this?
![grafik](https://user-images.githubusercontent.com/3872576/156989102-bd660a30-dc09-4a26-8577-e2bc8eb446a7.png)

I suspected that somehow I was not running v0.11, but this is exactly the version I see:
![grafik](https://user-images.githubusercontent.com/3872576/156988419-5ee653ed-560b-4f79-903d-731de6af18b0.png)

which is why I wrote: "how can I tell reliably... ". I'll reinstall the plugin and try again.
